clouddriver-cloudfoundry/clouddriver-cloudfoundry.gradle

```diff
@@ -51,15 +51,14 @@ dependencies {
   implementation "io.spinnaker.kork:kork-credentials"
   implementation "io.spinnaker.kork:kork-annotations"
   implementation "io.spinnaker.kork:kork-moniker"
+  implementation "io.spinnaker.kork:kork-web"
```

Comment:
Didn't `clouddriver-web` depend on `clouddriver-cloudfoundry`? Usually web module depends on all the others, not the other way around

Reply:
yes, clouddriver-web depends on clouddriver-cloudfoundry but not sure about kork-web. Either way, this was actually a mistake that slipped in during testing. We actually need to import micrometer directly because micrometer core isn't in kork. I think it was getting brought indirectly through kork-web because of the spring web dependencies.